web/html/src/components/table/TableDataHandler.tsx

```diff
@@ -284,13 +292,21 @@ export class TableDataHandler extends React.Component<Props, State> {
 
       const allSelected = currIds.length > 0 && currIds.every(id => selectedItems.includes(id));
       const checkbox = (
-        <Header key="check">
+        <Header key="check" width="30px">
           <input type="checkbox" checked={allSelected} onChange={e => handleSelectAll(e.target.checked)} />
         </Header>
       );
       headers && headers.unshift(checkbox);
     }
 
+    if (this.props.deletable) {
+      const deleteHeader = (
+        // Intentionally empty
+        <Header key="delete" width="30px" />
+      );
+      headers && headers.push(deleteHeader);
```

Comment:
Couldn't this be written as `headers?.push(deleteHeader)` ?

Reply:
Right now the type of `headers` is `false | JSX.Element[]`. It isn't a good option imo, but since that may affect other things in many other places I didn't want to go around changing it right now. Otherwise yes.